packages/paste-website/src/components/shortcodes/table-of-contents/TableOfContentsListItem.tsx

```diff
@@ -27,41 +27,18 @@ export const TableOfContentsListItem = styled.li<TableOfContentsListItemProps>`
   margin-top: ${marginTopPartial};
   font-size: ${fontSizePartial};
   line-height: ${lineHeightPartial};
-  color: ${themeGet('textColors.colorTextWeak')};
   border-radius: 2px;
 
-  a {
-    border-left-width: 2px;
-    border-left-style: solid;
-    border-left-color: transparent;
-
-    &:focus {
-      outline: none;
-    }
-  }
-
   &.is-current {
     font-weight: ${themeGet('fontWeights.fontWeightSemibold')};
-    color: ${themeGet('textColors.colorTextLink')};
 
     a {
+      color: ${themeGet('textColors.colorTextLink')};
       border-color: ${themeGet('borderColors.colorBorderPrimary')};
     }
   }
 
-  &:hover {
-    a {
-      border-color: ${themeGet('borderColors.colorBorderPrimaryStronger')};
-    }
-
-    color: ${themeGet('textColors.colorTextLinkStronger')};
-  }
-
```

Comment:
Just for my own understanding, all of this hover/focus behavior is now being specified in the specialized Anchor component, right? Why did we choose to transfer it there? And are we using TableOfContentsListItem in other places that would be affected by this change?

Reply:
We chose to move it there bc it keeps all the anchor styles in the same place. 

Yah! So now with this PR, all the places using TableOfContentsListItem are also using TableOfContentsAnchor, so the styles will still work.